Components/FlightControl/RocketSM.cpp

```diff
@@ -982,12 +982,9 @@ RocketState Test::OnEnter()
  */
 RocketState Test::OnExit()
 {
-	MEVManager::CloseMEV();
-	GPIO::MEV_EN::On();
-
 	osDelay(BURN_TIMER_PERIOD_MS);
```

Comment:
why do we have this delay before toggling the solenoid. It doesn't make sense to have a delay when switching states

Reply:
Ah yeah old artifact of the servo mev auto-close on exit of test